website/docs/learning-and-resources/how-to-guides/how-to-submit-multipart-and-binary-form-data.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Guide for submitting multipart/form-data and binary file data via Appsmith
+sidebar_position: 17
+---
+
+# Submitting Multipart and Binary form Data
+
+Appsmith supports sending file data in API queries via `multipart/form-data` encoding, which is commonly expected by API endpoints that handle file uploads. Multipart requests can include several different types of data within them, such as a file along with some other related metadata. The Appsmith REST API interface can also handle submitting raw binary file data.
```

Comment:
Put this under the Multipart form data section. Here write a general intro about send body data with API requests.